pgpy/pgp.py

```diff
@@ -2447,9 +2447,11 @@ def _preiter(first, iterable):
         loaded = set()
         for key in iter(item for ilist in iter(ilist if isinstance(ilist, (tuple, list)) else [ilist] for ilist in args)
                         for item in ilist):
-            if os.path.isfile(key):
+            if isinstance(key, PGPKey):
+                _key = copy.copy(key)
```

Comment:
Right, that's unnecessary. Maybe I saw some copying somewhere else in the library and thought this was a similar case. Will remove.